endoreg_db/views/anonymization_overview_views.py

```
@@ -0,0 +1,56 @@
+    permission_classes = DEBUG_PERMISSIONS   
+    pagination_class = NoPagination
+
+    def get_queryset(self) -> QuerySet:                  # type: ignore
+        qs_video = (
+            VideoFile.objects
+            .select_related("state")
+            .prefetch_related(
+                Prefetch(
+                    "label_video_segments",
+                    queryset=VideoFile.label_video_segments.rel.related_model.objects
+                             .filter(state__is_validated=True)
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)